source/docs/reference/environment-variables.md

```diff
@@ -0,0 +1,124 @@
+---
+id: environment-variables
+title: Environment variables
+---
+
+The environment variables available in Front-Commerce are the configurations that are likely to change depending on the current environment of your application. For instance, you could have three different environments: production, staging and local.
+
+These environment variables can be defined in two diferrent ways:
+* on your server (See [How To Read and Set Environmental and Shell Variables on a Linux VPS](https://www.digitalocean.com/community/tutorials/how-to-read-and-set-environmental-and-shell-variables-on-a-linux-vps))
+* in the `.env` file in your root folder
+
+You can then access them by using the `process.env` object in your javascript files. This variable is available in both your server side code and client side code. However,
```

Comment:
It could be misleading… here is a proposal:

```suggestion
You can then access them by using the `process.env` object in your javascript files no matter if it is a server-side or client-side file. However,
```